pkg/convert/pot_test.go

```diff
@@ -1005,7 +1005,7 @@ func TestFromPot(t *testing.T) {
 				Messages: []model.Message{
 					{
 						ID:          "Hello",
-						Message:     "{Hello, world!\nvery long string\n}",
+						Message:     "{\nHello, world!\\\\n\nvery long string\\\\n}",
```

Comment:
Why so many escapes?

Escaped `\n` is escaped as string literal `\\n` in golang.
Isn't `\n` escaped as`\\n` in n MF2, too?


```suggestion
						Message:     "{\nHello, world!\\n\nvery long string\\n}",
```

Reply:
Ok, after some thinking - `\n` should be escaped in `\\n` in MF2, right? Now, to translate it into golang string literal we have to escape both slashes - `\\\\n`.

Reply:
po has raw text:
`"Hello world\n"` (\n is not newline here, but \\\n)
after converting it to golang string it becomes
`"Hello world\\n"`
as message format tells us that in the text backslashes must be escaped it becomes:
`"Hello world\\\\n"`

Reply:
in one image
![image](https://github.com/translate-agent/translate/assets/69954124/2c87dd84-4b53-45a4-b1d6-6e9ecf98a2eb)